doc/input-output-reference/src/hvac-template-objects/hvactemplate-processing.tex

```diff
@@ -14,17 +18,88 @@ \section{HVACTemplate Processing}\label{hvactemplate-processing}
+\textbf{Users need to remember that no objects related to HVAC except for HVAC template objects are needed in the idf/epJSON file.} The existence of other objects (unless specifically described in the following sections) may cause unexpected errors to occur. \hyperref[sizingzone]{Sizing:Zone}, \hyperref[sizingsystem]{Sizing:System}, and \hyperref[sizingplant]{Sizing:Plant} objects will be generated by the corresponding HVACTemplate objects; the user does not need to create these elsewhere in the input file.
 
 There are some exceptions to this rule:
 
 \begin{itemize}
 \item
-  \hyperref[hvactemplateplantchillerobjectreference]{HVACTemplate:Plant:Chiller:ObjectReference} which requires that the corresponding chiller object be present in the idf file along with any required curve or performance objects. In this case, the HVACTemplate object does not create the chiller object, but adds all of the connections. \hyperref[hvactemplateplanttowerobjectreference]{HVACTemplate:Plant:Tower:ObjectReference} and \hyperref[hvactemplateplantboiler]{HVACTemplate:Plant:Boiler};ObjectReferences are similar.
+  \hyperref[hvactemplateplantchillerobjectreference]{HVACTemplate:Plant:Chiller:ObjectReference} which requires that the corresponding chiller object be present in the idf/epJSON file along with any required curve or performance objects. In this case, the HVACTemplate object does not create the chiller object, but adds all of the connections. \hyperref[hvactemplateplanttowerobjectreference]{HVACTemplate:Plant:Tower:ObjectReference} and \hyperref[hvactemplateplantboiler]{HVACTemplate:Plant:Boiler};ObjectReferences are similar.
+\item
+  For HVACTemplate:Zone:* objects, if Outdoor Air Method = \emph{DetailedSpecification}, then any referenced \hyperref[designspecificationoutdoorair]{DesignSpecification:OutdoorAir} and \hyperref[designspecificationzoneairdistribution]{DesignSpecification:ZoneAirDistribution} objects must be present in the idf/epJSON file.
+\item
+  For \hyperref[hvactemplatezonevav]{HVACTemplate:Zone:VAV} and \hyperref[hvactemplatezonedualduct]{HVACTemplate:Zone:DualDuct}, if a Design Specification Outdoor Air Object Name for Control is specified, then the referenced \hyperref[designspecificationoutdoorair]{DesignSpecification:OutdoorAir} object must be present in the idf/epJSON file.
+\end{itemize}
+
+\subsection{Command Line Interface}
+
+\subsubsection{Simulation Tool}
+
+The command line interface located in the EnergyPlus root directory (energyplus.exe) provides optional arguments to enable various features of the simulation package.  Below are the options relating to the ExpandObjects and pyExpandObjects processes.
+
+\subsubsection*{IDF Options}
+
+\begin{itemize}
+\item
+  -x --expandobjects : Run ExpandObjects prior to simulation
+\end{itemize}
+
+\subsubsection*{epJSON Options}\label{pyexpandobjects-cli-epjson-options}
+
+\begin{itemize}
```

Comment:
Please make it clear these are pyExpandObjects CLI options. Maybe this belongs in the next section.